### DISCUSSION TITLE
Pkinit certificate - SAN extensions

### DISCUSSION THREAD

**Discussion by davidrouss** [smallstep/certificates#1537] [Q&A]
Hello !! I need your help ! 

I need to create a pkinit certificate with these OIDs in SAN extensions:
![image](https://github.com/smallstep/certificates/assets/82118137/222d1fe4-6bce-4ba4-a6aa-9ae6e2cc8192)

I 'm not very familiar with asn1 and I tried many times without success..

Can you help me please ?? :) :)

**Comment by tashian** [comment]
The [templating system](https://smallstep.com/docs/step-ca/templates/) includes functions for generating ASN.1 values. You can use these functions in a template to meet your needs. Unfortunately we're a small team and we don't have the capacity to make a PKINIT template for you.
